mkdocs.yml

```diff
@@ -86,6 +62,23 @@ theme:
 
 ## Plugins
 plugins:
+  - redirects:
```

Comment:
did you had to install this plugin in your local environment?
If yes, we will need to add it to docs dependencies in `pyproject.toml` file ([here](https://github.com/up42/up42-py/blob/2b0d9fe605229b8a8d893c2f6ad464b0dd5782df/pyproject.toml#L52))

Reply:
Please add it, the publication fails at the moment